Dockerfile

```diff
@@ -18,4 +19,5 @@ COPY httpd.conf /etc/nginx/nginx.conf
 COPY --from=build /usr/src/app/dist .
 
 ENTRYPOINT ["/entrypoint.sh"]
-CMD ["nginx"]
\ No newline at end of file
+HEALTHCHECK --interval=8s --timeout=15s --start-period=120s --retries=128 CMD curl -f -s -o /dev/null 127.0.0.1
```

Comment:
with the included `wget` this would be `wget --quiet --tries=1 --spider --output-document=/dev/null 127.0.0.1` or `wget -q -t 1 --spider -O /dev/null 127.0.0.1`

Reply:
sure! fixed.